docs/source-1.0/spec/aws/aws-json.rst.template

```diff
@@ -192,6 +192,45 @@ In ``awsJson1_1``, servers SHOULD only send the error's
 for both protocols. See `Operation error serialization`_ for full details on how to
 deserialize errors for |quoted shape name|.
 
+.. smithy-trait:: aws.protocols#awsQueryError
+.. _aws.protocols#awsQueryError-trait:
+
+------------------------------------------
+``aws.protocols#awsQueryCompatible`` trait
+------------------------------------------
+
+Summary
+    When using ``awsQuery`` protocol, ```awsQueryError`` trait allows a custom code value defined for an error response.
+
+    When migrating from ``awsQuery`` to ``awsJson1_0`` protocol, ``awsQueryCompatible`` trait will continue this behaviour.
```

Comment:
I would also suggest to review this with someone who's native speaker, but for me is not clear which behaviour you're referring to here. I think it might be better to fully spell it out.